app/pb_kits/playbook/_playbook.scss

```diff
@@ -30,6 +30,7 @@
 @import 'pb_loading_inline/loading_inline';
 @import 'pb_message/message';
 @import 'pb_online_status/online_status';
+@import 'pb_nav/nav';
```

Comment:
You are including the nav 2 times. Also, make sure this is in alphabetical order

Reply:
👍